eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -357,6 +357,18 @@
           "new": "method void com.azure.spring.cloud.autoconfigure.aadb2c.AadB2cOidcLoginConfigurer::<init>(org.springframework.security.web.authentication.logout.LogoutSuccessHandler, org.springframework.security.oauth2.client.web.OAuth2AuthorizationRequestResolver, org.springframework.security.oauth2.client.endpoint.OAuth2AccessTokenResponseClient<org.springframework.security.oauth2.client.endpoint.OAuth2AuthorizationCodeGrantRequest>, org.springframework.boot.web.client.RestTemplateBuilder)",
           "justification": "New method added to fix a bug."
         },
+        {
+          "regex": true,
+          "code": "java.class.externalClassExposedInAPI",
+          "new": "interface jakarta\\.jms\\..*",
+          "justification": "They are external interfaces."
+        },
+        {
+          "regex": true,
+          "code": "java.class.externalClassExposedInAPI",
+          "new": "class jakarta\\.jms\\..*",
+          "justification": "They are external classes."
+        },
```

Comment:
Do we need this?

Reply:
Yes, I am trying to fix it. https://dev.azure.com/azure-sdk/public/_build/results?buildId=2055287&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=70ccc7c9-8d04-5255-456e-9568fe142bcd&l=13

Reply:
Removed.